main/variables.tf

```diff
@@ -203,3 +208,8 @@ variable "db-password-key" {
   description = "Key to acquire the database password for the strategy specified."
   type        = "string"
 }
+
+variable "cleanup-buckets" {
```

Comment:
plural what's the format? comma separated list of buckets? s3:// or without?

Reply:
comma separated list of bucket arns, will update the description

Reply:
ta